rpc-server/src/error.rs

```diff
@@ -79,6 +79,9 @@ pub enum Error {
     #[error("Multiple transactions found: {0}")]
     MultipleTransactionsFound(Blake2bHash),
 
+    #[error("Invalid argument: {0}")]
+    InvalidArgument(String),
```

Comment:
@jsdanielh out  of curiosity.

Is there a way to change this line to accept `&str` so we then avoid the use of `to_string()`

Reply:
You could change it to `&'static str`, but then it only works for string literals in the source code. Your current solution is more general.